docs/conf/user-interface/index.md

```diff
@@ -0,0 +1,621 @@
+      ),
+    },
+  ],
+  /* highlight-add-end */
+});
+```
+
+| Name       | Description                                                                                                                                                                                                                              |
+| ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`       | Each theme has a unique `id`                                                                                                                                                                                                             |
+| `title`    | This will be shown in the settings page to select the right theme.                                                                                                                                                                       |
+| `variant`  | This can be either `light` or `dark`. This is also referred to as `mode`. On the `body` of your app we are inserting a data attribute to set the theme based on this value: `data-theme-mode="light"`.                                   |
+| `icon`     | This will be shown in the settings page as a visual element to complement the title.                                                                                                                                                     |
+| `Provider` | This is needed to set the legacy theme with MUI only. This will be become redundant later on when we fully replace with BUI but for now you need to have it for MUI to work. BUI is based on CSS and don't rely on any global providers. |
+
+:::note
+Your list of custom themes overrides the default themes. If you still want to use the default themes, they are exported as `themes.light` and `themes.dark` from [`@backstage/theme`](https://www.npmjs.com/package/@backstage/theme). Be sure to provide both `light` and `dark` modes so users can choose their preference.
+:::
+
+## Create a theme for Backstage UI (New)
+
+Backstage UI is built entirely using CSS. By default we are providing a default theme that include all our core CSS variables and component styles. To start customising Backstage UI to match your brand you need to create a new CSS file and import it directly in `packages/app/src/App.tsx`. All styles declared in this file will override the default styles. As your file grow you can organise it the way you want or even import multiple files.
+
+Backstage UI is using light by default under `:root` but you can target it more specifically using the data attribute for mode
+
+```css title="packages/app/src/styles.css"
+:root {
+  /* Use :root to set styles for both light and dark themes */
+  .bui-Button {
+    background-color: #000;
+    color: #fff;
+  }
+}
+
+[data-theme-mode='light'] {
```

Comment:
Currently light just uses `:root`, do you think we should change that? I thought it was quite good since it removes the requirement to set the data attribute to have a functioning theme